lib/node_modules/@stdlib/math/base/special/acoth/benchmark/cpp/boost/benchmark.cpp

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
Boost does not have an `acoth` function. This file should be deleted.

You can verify this [here](https://github.com/boostorg/math/tree/develop/include/boost/math/special_functions).

Reply:
Thanks for the link! Will be useful for the next methods